README.md

```diff
@@ -211,6 +211,49 @@ This mixin binds the url of the best fitting image to the source attribute, base
 
 This mixin binds the url of the best fitting image as the background url to the elements style attribute, based in the values provided by the `image` and `size` attribute. It also get the `responsiveImage` service injected.
 
+##Extensibility hooks
+### Extend the image processing
+
+During the image process the addon calls the `preProcessImage` and the `postProcessImage` hooks for each origin image and supported width. Here you can add custom image process steps, like a watermark integration. The first hook will be called just before the addon's image process calls applies, the latter after. You can register your callbacks by calling the addon's `addImagePreProcessor` or `addImagePostProcessor` function before the addon's `postprocessTree` was called.
+In both cases the callback method you provide must have the following signature:
```

Comment:
I think an example how to get access to the addon instance would be useful!

Reply:
A few lines later I point to the new `ember-lazy-responsive-image` - addon as an example. For now it's empty. But the addon should be ready 'till the next days.